find-compatible-software-version.md

```diff
@@ -0,0 +1,90 @@
+## 如何找到适配自己电脑的软件版本
+
+看完这篇指南，你将学会如何在 GitHub 下载软件。
+
+[Windwos 用户点这里](#Windows用户下载以及安装)
+
+### MacOS用户下载以及安装
+
+点击屏幕左上方的  符号，在弹出来的菜单中，选择第一个选项 “关于本机（About This Mac）”
+
+<img src="find-compatible-software-version-IMG/1.png" alt="1" style="zoom:67%;" />
+
+在弹出来的窗口中，就可以看到本机的硬软件配置：
+
+<img src="find-compatible-software-version-IMG/2.png" alt="2" style="zoom: 50%;" />
+
+如果你看到的画面和上面的图片类似，说明你的 Mac 是配置了 Apple M 系列（M1、M2、M3）的芯片，这是 **ARM64 架构**的处理器，我们一会要下载 **ARM64 版本**的软件。
+
+如果看到的画面是下面这样的，有 **Intel 字样**：
+
+<img src="find-compatible-software-version-IMG/3.jpg" alt="2" style="zoom: 50%;" />
+
+说明你的 Mac 是 **x64 架构**，我们一会要下载 **x64 版本**的软件。
+
+点击[这里](https://github.com/xiaolai/everyone-can-use-english/tags)跳转到 [releases 页面](https://github.com/xiaolai/everyone-can-use-english/tags)下载软件。
+
+<img src="find-compatible-software-version-IMG/4.png" alt="2" style="zoom: 50%;" />
+
+注意看发布的日期，**越靠前的版本越新**，我们点击 “Downloads” 跳转到下载页面
+
+<img src="find-compatible-software-version-IMG/5.png" alt="2" style="zoom: 50%;" />
+
+我们要下载的软件就在 Asset 里，如果 Assets 像这样，没有展开：
+
+<img src="find-compatible-software-version-IMG/6.png" alt="2" style="zoom: 50%;" />
+
+可以点击 “ ▶ Assets ” 旁边的 “ ▶ ” 符号展开
+
+<img src="find-compatible-software-version-IMG/7.png" alt="2" style="zoom: 50%;" />
+
+如果你是 **ARM 架构**的 Mac，点击包含 “**darwin-arm64**” 字样的链接下载 Enjoy 程序；
+
+如果你是 **intel 架构**的 Mac，点击包含 “**darwin-x64**” 字样的链接下载 Enjoy 程序；
+
+下载完成后，双击这个下载好的 ZIP 文件，系统会自动解压
+
+<img src="find-compatible-software-version-IMG/8.png" alt="2" style="zoom: 50%;" />
+
+打开 “终端” 输入
+
+```bash
+sudo xattr -rd com.apple.quarantine 
+```
+
+
+> [!IMPORTANT]
+> 注意：`sudo xattr -rd com.apple.quarantine` 这行命令后面，**要加一个 “空格”**
```

Comment:
It could be a dangerous command for the newbie. Should we add some more explaination and potential risk of this command? It should never be used in any other software.

Reply:
Makes sense, I would add a warning: “Executing this command bypasses macOS’s security checks. Do not use this method to circumvent security checks in any other software, as it may lead to unknown security risks.” @an-lee